### PR TITLE
Throw standard exception on "callback never reached" error

### PR DESCRIPTION
**Pull request by mvorisek**
No need for late error - even if some output has been already sent, if will be detected by output writer

Why it is important: to be able to pass/display exception in json as any other to/in browser

image

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -1091,8 +1082,8 @@ protected function outputResponse(string $data, array $headers): void
             }
         }
 
-        if ($lateErrorStr !== null) {
-            echo $lateErrorStr;
+        if ($lateError !== null) {
+            echo "\n" . '!! ATK4 UI ERROR: ' . $lateError . ' !!' . "\n";
```
Comment (georgehristov) [inline src/App.php:1086]: I would change "ATK4 UI" to `$app->title` if possible here to enable white labeling of application or just remove it

**Review by georgehristov — APPROVED** [review]
Idea is very good, only one comment (unrelated but can be taken care of now)

File: src/App.php (hunk 1 above)
Comment (mvorisek) [reply src/App.php:1086]: This is fatal error that should never apper in production.

Comment (mvorisek) [reply src/App.php:1086]: We already have simillar error in core for exception renderer error.

**Pull request merged**